inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -271,6 +271,18 @@
         </where>
     </select>
 
+    <update id="updateByGroupIdAndStreamId">
```

Comment:
It is recommended to modify the method name, indicating that this method is a delete operation

Reply:
This is a logical delete, maybe `update` would be better.